tools/docs/docs_generate.ts

```diff
@@ -27,19 +27,55 @@ const componentsFolder = './src';
 const inheritedFromColumnIndex = 6;
```

Comment:
I wonder if this could be solved by a jsdoc annotation on the class?
However, I am not opposed to this solution.

Reply:
I created an [issue](https://github.com/orgs/sbb-design-systems/projects/4/views/1?pane=issue&itemId=81059568) in the next column to further develop/refactor the doc override solution.
I did not want to delay this PR further 😔